docs/index.html

```diff
@@ -3646,21 +3645,13 @@ <h2>Vocabulary</h2>
                 "reporting-exception": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/direct-parent-reporting-exception"
               }
             },
-            "ultimate-parent": {
-              "links": {
-                "reporting-exception": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/ultimate-parent-reporting-exception"
-              }
-            },
-            "direct-children": {
-              "links": {
-                "relationship-records": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/direct-child-relationship",
-                "related": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/direct-children"
-              }
-            },
-            "isins": {
-              "links": {
-                "related": "https://api.gleif.example.org/api/v1/lei-records/8W58S473UJBX3RDKQDJN/isins"
-              }
+            "https://schema.org/unitCode": {
```

Comment:
I think this change is likely to get overwritten every time the spec is regenerated.

Reply:
We generate the JSON, embed it in respec, and then render respec at runtime via javascript.